### PR TITLE
Refactor stats to work with EA.org

### PR DESCRIPTION
**Pull request by colophonemes**

### CODE REVIEW

**Comment by jpaddison3** [comment]
Invalid rank_by_pledge_type?

**Comment by jpaddison3** [comment]
I was also getting a "Our *...* members have..." (first ellipsis literal) which I may have mentioned to you and is what caused me to assign the review to you

File: tasks/metalsmith.js [hunk 1]
```diff
@@ -190,12 +190,15 @@ function build (buildCount) {
...
-        // hack to add stats for AMF
+      // hack to add stats for AMF
       var meta = metalsmith.metadata()
       meta.stats.amfCostPerLifeSaved = 2838
-      // meta.stats.numberMembers = meta.stats.TotalPledges
-      // meta.stats.amountDonated = meta.stats.TotalPayments
-      // meta.stats.amountPledged = meta.stats.TotalLiability
+      meta.stats.numberMembers = meta.stats.TotalGWWCMembers
+      // hardcode numbers from database
+      // parfit=> SELECT sum(amount_normalized)::MONEY from pledges.reported_donation;
```
Comment (jpaddison3) [inline tasks/metalsmith.js]: What's all the dead code?

File: tasks/stats/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  // 'numberofmembers',
+  // 'amountpledged',
+  // 'amountpledgedwords',
+  // 'amountdonatedsofar',
+  // 'igivechart',
+  // 'igivechartwithoutother',
+  // 'citieschart',
+  // 'chaptermap',
+  // 'listofmembers'
```
Comment (jpaddison3) [inline tasks/stats/index.js]: What's this?

**Comment by jpaddison3** [comment]
Stats seem to be calculating correctly, but displaying wrong.

**Comment by jpaddison3** [comment]
Members list looks good

File: tasks/metalsmith.js (hunk 1 above)
Comment (colophonemes) [reply tasks/metalsmith.js]: This is just to leave a clue how to get this number until we add completion payments and do it properly

File: tasks/stats/index.js (hunk 2 above)
Comment (colophonemes) [reply tasks/stats/index.js]: These are the names of the stats from the old Drupal site. I could probably kill them.

**Comment by colophonemes** [comment]
> Stats seem to be calculating correctly, but displaying wrong.

Screenshot? Is this the issue where the member count on the frontpage is `...` and if you look at the code for that element, it says 'data-numbermembers=0`? If so, that's a definite bug. If not, then I'll need to do more digging.

File: tasks/stats/index.js (hunk 2 above)
Comment (jpaddison3) [reply tasks/stats/index.js]: Document it or kill it

**Comment by jpaddison3** [comment]
image

**Comment by jpaddison3** [comment]
<span class="number numberMembers" data-number="1" id="amazing-thing-number-0">...</span>

**Comment by colophonemes** [comment]
Can you try adding a couple more GWWC members to your DB? It works fine for me, I suspect the issue is something like the rolling-numbers animation starting at `1`, but then there's only `1` member to get to, so it never gets going.

**Pull request merged**